protocol/testing/genesis.sh

```diff
@@ -1012,13 +1019,17 @@ function edit_genesis() {
 		next_bank_idx=$(($next_bank_idx+1))
 	fi
 
-	# Initialize bank balance for reward vester account.
-	dasel put -t json -f "$GENESIS" ".app_state.bank.balances.[]" -v "{}"
-	dasel put -t string -f "$GENESIS" ".app_state.bank.balances.[$next_bank_idx].address" -v "${REWARDS_VESTER_ACCOUNT_ADDR}"
-	dasel put -t json -f "$GENESIS" ".app_state.bank.balances.[$next_bank_idx].coins.[]" -v "{}"
-	dasel put -t string -f "$GENESIS" ".app_state.bank.balances.[$next_bank_idx].coins.[0].denom" -v "${REWARD_TOKEN}"
-	dasel put -t string -f "$GENESIS" ".app_state.bank.balances.[$next_bank_idx].coins.[0].amount" -v "10000000$EIGHTEEN_ZEROS" # 10 million full coins
-	next_bank_idx=$(($next_bank_idx+1))
+	if [ $(echo "$REWARDS_VESTER_ACCOUNT_BALANCE > 0" | bc -l) -eq 1 ]; then
```

Comment:
Can you run `localnet` and check that `localhost:26657/genesis` contains the correct balance for reward vester account?

Reply:
confirmed:
`{"address":"dydx1ltyc6y4skclzafvpznpt2qjwmfwgsndp458rmp","coins":[{"denom":"adv4tnt","amount":"200000000000000000000000000"}]}`